.travis.yml

```diff
@@ -1,22 +1,33 @@
+os: linux
 language: go
-sudo: false
 go:
-- 1.13.x
-- 1.12.x
-
+  - 1.13.x
```

Comment:
Probably want to have 1.14.x and 1.13.x and drop 1.12.x now (I know, PR was originally created long ago).

Reply:
fixed

Reply:
fixed